portfolio/src/main/webapp/script.js

```diff
@@ -33,10 +33,68 @@ async function getContent() {
   document.getElementById('message-container').innerText = message;
 }
 
+function postComments(comments){
+    console.log(comments);
+    const messageBoard = document.getElementById('comments-container');
+    comments.forEach((comment) => {
+      messageBoard.appendChild(createCmtEl(comment));
+    }) 
+}
+
+function deleteAllComments(){
+    fetch('/delete-data',{method:"POST"}).then(response => response.json()).then((emptyComments) => {
+        console.log(emptyComments);
+    });
+    getComments();
+}
+
+function deleteComment(){
+    fetch('/delete-data?comment-key='+getCommentKey()).then(response => response.json()).then((emptyComments) => {
```

Comment:
Could you please add a confirmation box before sending this request to the server. This will ensure that we don't delete the data, when someone accidently clicks on that button.